test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -285,3 +286,49 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 		})
 	})
 }
+
+// WaitForSnapshotReady waits for a VolumeSnapshot to be ready to use or until timeout occurs, whichever comes first.
+func WaitForSnapshotReady(c dynamic.Interface, ns string, snapshotName string, poll, timeout time.Duration) error {
```

Comment:
This is copied from test/e2e/storage/utils/snapshot.go?  It is not called anywhere and should be removed.